src/common/webrequest_curl.cpp

```diff
@@ -104,6 +184,27 @@ size_t wxWebResponseCURL::CURLOnHeader(const char * buffer, size_t size)
     return size;
 }
 
+int wxWebResponseCURL::CURLOnProgress(curl_off_t total)
+{
+    if ( m_knownDownloadSize != total )
+    {
+        if ( m_request.GetStorage() == wxWebRequest::Storage_Memory )
+        {
+            SetBufferSize(static_cast<size_t>(total));
+        }
+        m_knownDownloadSize = total;
+    }
+
+    if ( static_cast<wxWebRequestCURL&>(m_request).HasPendingCancel() )
+    {
+        return 1;
+    }
+    else
+    {
+        return wxWebSessionCURL::ProgressFuncContinue();
```

Comment:
```suggestion
        return 0;
```
This change should make everything work without having to change `STDERR` option.

Reply:
This change was made in 3e5fd54.